### ISSUE TITLE
How to share content along with an image using this library?

### ISSUE THREAD

**Issue by bharat787** [harshzalavadiya/react-web-share#17]
Trying to share on socials with a image included. Is there a way to achieve this? Shared an example below.

![WhatsApp Image 2021-05-20 at 10 11 03 AM](https://user-images.githubusercontent.com/40318290/118920269-bb41ae00-b953-11eb-95a6-822dc02ddfbe.jpeg)

**Comment by harshzalavadiya** [comment]
@bharat787 I don't think that image sharing is possible for this plugin since it's design to share URL's not images, sharing images are complete different thing

**Comment by bharat787** [comment]
@harshzalavadiya I supposed that the underlying workings were similar to [Web Share API](https://web.dev/web-share/ ) which does allow to transfer files/images. Can this be added in the future?
Anyways, thanks for the plugin, it works great!

**Comment by sam4rano** [comment]
I just hope image sharing is added, it's really a nice package but should also be improved to share image

**Comment by kan2k** [comment]
+1 for this feature

**Comment by Ibby-debug** [comment]
+1
